src/components/Table/TableBase.tsx

```diff
@@ -450,6 +450,7 @@ export class TableBase<T> extends React.Component<
 			onRowClick,
 			getRowHref,
 			getRowClass,
+			onSort,
```

Comment:
Let's not destructure an object property that we don't use. To solve this problem, I would just remove the `{...props}` from the Base component, since it only accept the `InternalTableBaseProps` a these are already passed to the component. This will probably also allow you to also remove the `...props`  right below 

```
interface InternalTableBaseProps {
	hasCheckbox: boolean;
	hasRowClick: boolean;
	hasGetRowRef: boolean;
}
```